packages/docusaurus-plugin-client-redirects/package.json

```diff
@@ -18,18 +18,20 @@
   },
   "license": "MIT",
   "dependencies": {
-    "@docusaurus/core": "2.0.0-beta.9",
-    "@docusaurus/types": "2.0.0-beta.9",
-    "@docusaurus/utils": "2.0.0-beta.9",
-    "@docusaurus/utils-common": "2.0.0-beta.9",
-    "@docusaurus/utils-validation": "2.0.0-beta.9",
+    "@docusaurus/core": "workspace:*",
```

Comment:
I guess this is only for testing purposes?

Reply:
No, this is how things are done in Yarn 3.  Putting an actual version would make it fetch from the npm registry.  Plus, this way, the release diff is smaller.

Reply:
We use lerna to automate the release (yes, a weird mix of yarn workspaces and lerna). In the perfect world we ditch lerna altogether in this PR, but otherwise we have to make sure they interop

Reply:
> Putting an actual version would make it fetch from the npm registry

NIT: Only if the range doesn't match the version of the workspace

Reply:
Wait, @merceyz Does that mean we can keep using the actual version (`2.0.0-beta.9`) in the workspace and publish with lerna?

Reply:
Actually—no, let's not get stuck on the good old lerna forever. @slorber What's your call? Should we migrate to using Yarn workspaces throughout or should we continue using lerna, provided they would have similar DX and no user-side differences?